typings/generated/entities/organization.d.ts

```diff
@@ -0,0 +1,33 @@
+import { AxiosInstance } from 'axios';
+import { MetaSysProps, CollectionProp } from '../common-types';
+export declare type OrganizationProp = {
+    /**
+     * System metadata
+     */
+    sys: MetaSysProps;
+    /**
+     * Name
+     */
+    name: string;
+};
+/**
+ * This method creates the API for the given organization with all the methods for
+ * reading and creating other entities. It also passes down a clone of the
+ * http client with an organization id, so the base path for requests now has the
+ * organization id already set.
+ * @private
+ * @param  {Object} http - HTTP client instance
```

Comment:
Looks like it needs to be regenerated

Reply:
@burakukula please, run `build:types` one more time

Reply:
upsi, forgot about it. ✅